content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -22,7 +22,8 @@ All **CronJob** `schedule:` times are based on the timezone of the master where
 {{< /note >}}
 
 When creating the manifest for a CronJob resource, make sure the name you provide
-is no longer than 52 characters. This is because the CronJob controller will automatically
+is a valid [DNS subdomain name](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names).
```

Comment:
@tengqm , Does CronJob follow the DNS label or DNS subdomain name spec?
Is it possible to point to the individual sections in the spec (https://tools.ietf.org/html/rfc1123)?

Reply:
The validation of CronJob names is done [here](https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/batch/validation/validation.go#L177-L186)

The validator reuses [the name validation logic for replication controllers](https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/validation/validation.go), which is redirected to *DNS Subdomain* check.
It then checks if the name is too long because cronjob name specified is just a prefix.

Reply:
Pointer to the RFC spec is given in the overall DNS subdomain constraint, so we don't need to repeat that every time we reference it.

Reply:
👍